cdat/cdat/DictEdit/DictEditor.py

```diff
@@ -154,27 +150,19 @@ def checkKeyValues(self):
         for row in self.key_value_rows:
             l_text = row.key()
             r_text = row.value()
-            if l_text in keys or l_text == "" or r_text == "":
-                return (False, False)
-
-            keys.append(l_text)
-            values.append(r_text)
-
+            if l_text not in keys and l_text != "":
```

Comment:
Also, you can just do `if l_text and l_text not in keys` (empty strings evaluate to false)